Include/Ashkal/Shapes/Stage.hpp

```diff
@@ -11,11 +11,11 @@ namespace Ashkal {
       Stage() = default;
 
       //! Adds a shape at the specified Point.
-      void add_shape(std::unique_ptr<Shape> shape, const Point& point);
+      void addShape(std::unique_ptr<Shape> shape, const Point& point);
```

Comment:
Functions are snake_case.

Reply:
I thought we changed that so that variable names don't clash with Class and function names?